lib/models/messages/twitch/message.dart

```diff
@@ -161,6 +161,7 @@ Iterable<MessageToken> rootEmoteTokenizer(String message, String emotes) sync* {
 class TwitchMessageModel extends MessageModel {
   final TwitchUserModel author;
   final String message;
+  final Map<String, dynamic>? reply;
```

Comment:
Can we parse `reply` into a typed object instead of leaving it as a map?

Something like

```
class TwitchMessageReplyModel {
  final String messageId;
  final String message;
  final TwitchUserModel author;
}
```

Reply:
Done!